task-sdk/tests/task_sdk/bases/test_operator.py

```diff
@@ -711,7 +711,7 @@ def fn_to_template(**kwargs):
         task.render_template_fields({})
         assert task.arg2 == "foo_barbarbar"
 
-    @pytest.mark.parametrize(("content",), [(object(),), (uuid.uuid4(),)])
+    @pytest.mark.parametrize("content", [object(), uuid.uuid4()])
```

Comment:
Double-check this one for me?  It feels wrong.  It was a tuple and now it isn't, but then you modified the values too?

Reply:
<img width="748" height="232" alt="截圖 2025-11-06 凌晨1 46 53" src="https://github.com/user-attachments/assets/6ff9f404-87a6-470e-a04a-5ae0d35bc6cb" />
ruff check said that it should be a str

Reply:
hm.   Maybe try this:

```suggestion
    @pytest.mark.parametrize(("content",), [object(), uuid.uuid4()])
```

Reply:
<img width="625" height="71" alt="截圖 2025-11-06 凌晨2 06 08" src="https://github.com/user-attachments/assets/c952af15-7a05-47b4-b415-900bf130f52a" />

pytest failed. I have check PT006 rule, it said single param should be str.

Reply:
I have check the pytest src. The result is different because the force_tuple param. The detail is explained in the PR you mentioned.